presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaPlugin.java

```diff
@@ -30,12 +29,16 @@
 public class KafkaPlugin
         implements Plugin
 {
+    public static final Module DEFAULT_EXTENSION = binder -> {
+        binder.install(new KafkaConsumerModule());
```

Comment:
I wonder if it wouldn't be better to model kafka consumer as optional binding, see how I implemented Glue column stats extension, `GlueMetastoreModule`.

Reply:
actually it is not optional. To make kafka work one has to provide kafka consumer. By it could be a different class depending on how kafka is extended.